pkg/upgrade/upgrader/jiva_upgrade.go

```diff
@@ -411,6 +409,20 @@ func (j *jivaVolumeOptions) preupgrade(pvName, openebsNamespace string) error {
 		return err
 	}
 
+	if currentVersion < "1.9.0" {
+		klog.Infof("Scaling down old deployments")
+		err = j.preMigration(openebsNamespace)
+		if err != nil {
```

Comment:
in case of failures, do we need to revert the partial upgrade that has been done so that operations go fine? or would that be out of scope of upgrade jobs?
Did we handled such things like reverting partial upgrade in any previous failures?
is this scope for higher level operator (or) need to follow blue-green deployments?